frontend/lib/src/components/auth/auth_header.dart

```diff
@@ -92,10 +93,13 @@ class AuthHeader extends StatelessWidget {
                         child: FutureBuilder(
                           future: controller.getFullName(),
                           builder: (context, snapshot) {
-                            return LogoutDropdown(
-                              name: controller.name,
-                              value: 'logout',
-                              onChanged: dropdownCallback,
+                            return Obx(
+                              () => LogoutDropdown(
+                                name:
+                                    truncate(controller.name.toUpperCase(), 20),
```

Comment:
this will look weird on wider screens, dont limit the display by a length,
previous implementation was ok